dist/index.html

```diff
@@ -1 +1,40 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>WebPack Exercise</title><script defer="defer" src="main.js"></script></head><body><h1>Hello WebPack!</h1></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mini ToDo List</title>
+<script defer src="main.js"></script></head>
+<body>
+  <div class="container">
+    <ul id="tdlist">
+      <li class="title">
+        <h1>Today's To Do</h1>
+        <span class="material-symbols-outlined icon">sync</span>
+      </li>
+      <li class="add">
+        <span class="input">Add to your list...</span>
+        <span class="material-symbols-outlined icon">add_task</span>
+      </li>
+      <li class="task">
+        <span class="material-symbols-outlined unchecked">radio_button_unchecked</span>
+        <span class="item">Wash the dishes</span>
+        <span class="material-symbols-outlined icon">more_vert</span>
+      </li>
+      <li class="task">
+        <span class="material-symbols-outlined unchecked">radio_button_unchecked</span>
+        <span class="item">Complete the project</span>
+        <span class="material-symbols-outlined icon">more_vert</span>
+      </li>
+      <li class="btn-sweep">
+        <span>Clear all completed</span>
+        <span class="material-symbols-outlined icon">delete_sweep</span>
+      </li>
+      <!-- <li class="title"><span class="material-symbols-outlined">task_alt</span></li>
+        <li class="title"></li>
+        <li class="title"><span class="material-symbols-outlined">delete</span></li> -->
```

Comment:
- [ ] Good job so far :+1: Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

Reply:
Thanks for the feedback @Owusu-Desmond 👍.
I've already asked for another CR.
Cheers,
Luis 👋